CHANGELOG.md

```diff
@@ -2512,9 +2512,11 @@ Swift 2.2
     AnyObject>`.  It bridges in O(1), without memory allocation.
 
   - `Dictionary<K, V>` has an implicit conversion to `NSDictionary`.
-    `Dictionary<K, V>` bridges to `NSDictionary` iff both `K` and `V` are
+    `Dictionary<K, V>` bridges to `NSDictionary` [iff] both `K` and `V` are
```

Comment:
Hm, it's not valid [original-Markdown](http://daringfireball.net/projects/markdown/syntax#link), but it is in CommonMark (and clearly in GitHub-Flavored Markdown). I'd still prefer using the "collapsed" form with the empty `[]`, though, please!